tests/Client.UiTests/WebElementHelper.fs

```diff
@@ -0,0 +1,11 @@
+﻿module Client.UiTests.WebElementHelper
+
+open OpenQA.Selenium
```

Comment:
is it a new package? or is it part of canopy/selenium driver?

Reply:
Part of canopy stuff, did not have to install anything extra for that. Needed for that `By.TagName` thing.